Update README.md (#282)
citation-file-format/citation-file-format: README.md

> # Citation File Format (CFF)

Reviewer: Abbreviation shouldnt be part of the title
```suggestion
# Citation File Format
```

[![Build Status](https://github.com/citation-file-format/citation-file-format/workflows/testing/badge.svg)](https://github.com/citation-file-format/citation-file-format/actions/workflows/testing.yml)
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1003149.svg)](https://doi.org/10.5281/zenodo.1003149)
[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)

> The Citation File Format (CFF) is a human- and machine-readable file format in YAML 1.2 which provides

Reviewer: contracting the human and machine readability is a bit of an antipattern for English. Maybe we can end the sentence with something like _readable by humans as well as machines_, like so? 

```suggestion
The Citation File Format specifies how users can provide citation metadata that is easy to read by humans as well as by machines.
```

Author: ```suggestion
The Citation File Format specifies how users can provide citation metadata for software and datasets that is easy to read by humans as well as by machines.
```

citation metadata for software and datasets.

## Structure

> `CITATION.cff` files in the Citation File Format consist of three parts:

Reviewer: this section now reads as if you need to have these 3 parts

1. The **software** (or dataset) **citation metadata** - as the most important part, this is **required**.
2. A **references** section - works like a references section in a paper, so that you can give credit to the authors of works that your software or dataset builds on.

> 3. One **preferred citation** - so that even when you ask people to cite your software or dataset, they can also additionally cite a traditional paper.

Reviewer: explanation could be a bit more to the point, maybe something like

```suggestion
3. One **preferred citation** - when you want to redirect any credit your work may receive towards a second work (typically one of your own).
```

### Example

This is what a common `CITATION.cff` file for research software may look like:

```yaml
cff-version: 1.2.0
message: If you use this software, please cite it using these metadata.
title: My Research Software
abstract: This is my awesome research software. It does many things.
authors:
  - family-names: Druskat
    given-names: Stephan
    orcid: "https://orcid.org/0000-0003-4925-7248"
version: 0.11.2
date-released: "2021-07-18"
identifiers:
  - description: This is the collection of archived snapshots of all versions of My Research Software
    type: doi
    value: "10.5281/zenodo.123456"
  - description: This is the archived snapshot of version 0.11.2 of My Research Software
    type: doi
    value: "10.5281/zenodo.123457"
license: Apache-2.0
repository-code: "https://github.com/citation-file-format/my-research-software"
```

## Format specifications :books:

You can find the complete format specifications in the [Guide to the Citation File Format schema](schema-guide.md).

## Why should I add a `CITATION.cff` file to my repository? :bulb:

When you do this, great things may happen:

> 1. People that find your software can easily cite it using the correct metadata from `CITATION.cff`!

Reviewer: I know that GitHub was planning to have improved Search Engine Optimization based on metadata from `CITATION.cff`, not sure if that's already in production but that would also be a good motivator

Author: Too early.

> 2. If your repository is hosted on GitHub, [they will link to it](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files) in a special "Cite this repository" widget on the repository landing page. Also, the citation information in `CITATION.cff` will be used when you publish your software on Zenodo via the [GitHub-Zenodo bridge](https://guides.github.com/activities/citable-code/).

Reviewer: - "they will link to it" not a convincing argument, maybe something like _they will show a widget in the sidebar that makes it easy for visitors to cite your software correctly_
- github-zenodo bridge part could be a separate point

3. The citation information in `CITATION.cff` can be used by the open source reference manager [Zotero](https://github.com/zotero/zotero), when people import the repository.

## Validation :heavy_check_mark:

## Tools to work with `CITATION.cff` files :wrench:

> ## Maintainers :nerd_face:

Author: Hi @jspaaks, I think I'm finished with the table for now (i.e. https://github.com/citation-file-format/citation-file-format/blob/e6ffd6f1e6fde4dc99f62c3b8c56080ff0937d0a/README.md#tools-to-work-with-citationcff-files-wrench). Can you let me know what you think/if there is anything missing?

Will delete any empty rows once you've gone over it.

> ## Maintainers :nerd_face:

Reviewer: Looks OK. Visually it's a bit cluttered though, perhaps there is a way to equal-width the columns? Might require a plain HTML table. Also I prefer to have nouns as column headers, and I would merge Update into Edit. Are there empty cells now for which a suggestion has been made already as `tooling`? If so we could link to it

BTW this is already goog to merge IMO, we can iterate the content later.

> ## Maintainers :nerd_face:

Author: Thanks for the review. I'll 
- try to equal width the cols
- merge Update and Edit
- see if I can find something useful in the issues
- try nouns

> ## Maintainers :nerd_face:

Author: CSS styling is not possible in HTML in GitHub Markdown it seems (or is rather just ignored), so I reverted to a Markdown table. Should be good enough.

The Citation File Format schema is maintained by

- [Stephan Druskat](https://sdruskat.net) ([@sdruskat](https://github.com/sdruskat/))
- Jurriaan Spaaks ([@jspaaks](https://github.com/jspaaks/))

## Contributing :handshake:

**The _Citation File Format_ is a collaborative project** and we welcome suggestions and contributions. We hope one of the invitations below works for you, but if not, please let us know!

:running: **I'm busy, I only have 1 minute**
- Tell a friend about the *Citation File Format*!

:hourglass_flowing_sand: **I've got 5 minutes - tell me what I should do**
- Create a `CITATION.cff` file for your repository.

> - Suggest ideas for how you would like to use the *Citation File Format*, or for an improvement to the format or its tooling.

Reviewer: This is a bit risky, because it will likely take more than 5 minutes, and then we will have betrayed the trust that people put in this project the very first time we met

:computer: **I've got a few hours to work on this**
- Take a look at the issues and see if there are any you can contribute to.

> :tada: **I really want to help increase the community**

Reviewer: ```suggestion
:tada: **I want to help increase the community**
```

Author: Done.

- Organise a hack event or workshop to help others use or improve the *Citation File Format*.

Please read the more detailed [contributing guidelines](CONTRIBUTING.md) and [open a GitHub issue](https://github.com/citation-file-format/citation-file-format/issues) to suggest a new idea or let us know about bugs.

## License :balance_scale:

Copyright © 2016ff. The Citation File Format Contributors

> This work is licensed under multiple licenses:

Reviewer: Not sure that's technically true, but different parts of the repo are licensed differently. Maybe we should write down somewhere why we chose to have the mix, because I've forgotten ...again

Also the Zenodo metadata for 1.1.0 has yet a different license (CC-BY-SA-4.0; #245)
